components/movementsensor/cameramono/cameramono.go

```diff
@@ -290,10 +290,11 @@ func (co *cameramono) AngularVelocity(ctx context.Context, extra map[string]inte
 // Properties gets the position of the moving object calculated by visual odometry.
 func (co *cameramono) Properties(ctx context.Context, extra map[string]interface{}) (*movementsensor.Properties, error) {
 	return &movementsensor.Properties{
-		PositionSupported:        true,
-		OrientationSupported:     true,
-		AngularVelocitySupported: true,
-		LinearVelocitySupported:  true,
+		PositionSupported:           true,
+		OrientationSupported:        true,
+		AngularVelocitySupported:    true,
+		LinearVelocitySupported:     true,
+		LinearAccelerationSupported: true,
```

Comment:
Linear acceleration is not supported on this sensor: it gives back the unimplemented error if you try. Undo this change.